Dockerfile

```diff
@@ -57,5 +69,17 @@ RUN ./docker/build --backend
 
 RUN sha256sum /backend.wasm.gz
 
+
+ARG network="staging"
```

Comment:
Would it make sense to introduce a new build stage for the front-end build? As is, this is part of the `build_backend` stage, which now is probably no longer named accurately?

Reply:
Its in a different file now 👍